src/transactions/cdp.js

```diff
@@ -727,6 +734,8 @@ export async function closeCDP(accountID, microRepayGARD, payFee = true) {
   let response = await sendTxn(stxns, "Successfully closed your cdp with ID " + accountID + ".",);
   setLoadingStage(null)
   removeCDP(info.address, accountID);
+  updateLiquidationFirestore(accountID)
```

Comment:
Closing a CDP doesn't count as liquidation. This function should be called inside the liquidate() function instead